blazor/autocomplete/filtering.md

```diff
@@ -210,4 +210,17 @@ The AutoComplete component filter queries can be customized. You can also use yo
         await autoObj.FilterAsync(Country, query);
     }
 }
-```
\ No newline at end of file
+```
+
+## Custom Filtering with Custom Adaptor
+
+The following sample code demonstrates implementing custom filtering with custom data adaptor, which allows to perform manual operations on the data. This can be utilized for implementing custom data binding and editing operations in the AutoComplete component.
```

Comment:
The description and use of the solution is not clearly explained. Please modify it